src/vendor/github.com/gocraft/work/worker_pool.go

```diff
@@ -199,6 +199,11 @@ func (wp *WorkerPool) Start() {
 	wp.periodicEnqueuer.start()
 }
 
+// GetPoolID return the pool ID
+func (wp *WorkerPool) GetPoolID() string {
+	return wp.workerPoolID
+}
+
```

Comment:
DO NOT EDIT files in vendor manually, please update them through go.mod and go.sum.